src/Avatar/Avatar.stories.tsx

```diff
@@ -0,0 +1,49 @@
+import { Flex } from "../Flex";
+import { Avatar } from "../Avatar";
+
+export default { title: "Avatar" };
+
+export const Default = () => {
+	return (
+		<Flex>
+			<Avatar />
+			<Avatar
+				name="Sara"
+				src="https://media.istockphoto.com/photos/smiling-young-woman-beauty-close-up-portrait-picture-id1280113805?k=20&m=1280113805&s=612x612&w=0&h=4axzFdjwxVoxSCYGDZGgx0ipyhmOd7N4_OEyokFcZiU="
+			/>
+		</Flex>
+	);
+};
+
+export const AllSizesAvatar = () => {
+	return (
+		<Flex>
+			<Avatar
+				size="sm"
+				name="Sara"
+				src="https://media.istockphoto.com/photos/smiling-young-woman-beauty-close-up-portrait-picture-id1280113805?k=20&m=1280113805&s=612x612&w=0&h=4axzFdjwxVoxSCYGDZGgx0ipyhmOd7N4_OEyokFcZiU="
+			/>
+			<Avatar
+				name="Sara"
+				src="https://media.istockphoto.com/photos/smiling-young-woman-beauty-close-up-portrait-picture-id1280113805?k=20&m=1280113805&s=612x612&w=0&h=4axzFdjwxVoxSCYGDZGgx0ipyhmOd7N4_OEyokFcZiU="
```

Comment:
all this links are repeated, store it in constant in same file. So that it becomes easy to change it in future